src/main/java/org/launchcode/techjobs/oo/Job.java

```diff
@@ -2,24 +2,132 @@
 
 import java.util.Objects;
 
-public class Job {
+//    private int id;
+//    private static int nextId = 1;
+//
+//    private String name;
+//    private Employer employer;
+//    private Location location;
+//    private PositionType positionType;
+//    private CoreCompetency coreCompetency;
+
+    // TODO: Add two constructors - one to initialize a unique ID and a second to initialize the
+    //  other five fields. The second constructor should also call the first in order to initialize
+    //  the 'id' field.
+
+    // TODO: Add custom equals and hashCode methods. Consider two Job objects "equal" when their id fields
+    //  match.
+
+
+    // TODO: Add getters for each field EXCEPT nextId. Add setters for each field EXCEPT nextID
+    //  and id.
+import java.util.Objects;
 
+public class Job {
+    //fields
     private int id;
     private static int nextId = 1;
+    private static final String DEFAULT_MESSAGE = "Data not available";
 
     private String name;
     private Employer employer;
     private Location location;
     private PositionType positionType;
     private CoreCompetency coreCompetency;
 
-    // TODO: Add two constructors - one to initialize a unique ID and a second to initialize the
-    //  other five fields. The second constructor should also call the first in order to initialize
-    //  the 'id' field.
+    //getters
+    public int getId() {
+        return id;
+    }
+    public String getName() {
+        return name;
+    }
+    public Employer getEmployer() {
+        return employer;
+    }
+    public Location getLocation() {
+        return location;
+    }
+    public PositionType getPositionType() {
+        return positionType;
+    }
+    public CoreCompetency getCoreCompetency() {
+        return coreCompetency;
+    }
```

Comment:
```suggestion
        //constructors
    public Job(){
        id = nextId;
        nextId++;
    }
    
    public Job(String aName, Employer aEmployer, Location aLocation, PositionType aPositionType, CoreCompetency aCoreCompetency){
        this();
        name = aName;
        employer = aEmployer;
        location = aLocation;
        positionType = aPositionType;
        coreCompetency = aCoreCompetency;
    }
```
I would recommend moving your constructors above your getters and setters due to it being a common order for them that I've seen so far (i.e. Class Fields - 1st, Constructor(s) - 2nd). I think getters and setters can follow the methods or constructor, though.